packages/logger/src/Logger.ts

```diff
@@ -275,7 +275,7 @@ class Logger extends Utility implements ClassThatLogs {
    * @returns {HandlerMethodDecorator}
    */
   public injectLambdaContext(options?: HandlerOptions): HandlerMethodDecorator {
-    return (target, _propertyKey, descriptor) => {
+    return (_target, _propertyKey, descriptor) => {
```

Comment:
Housekeeping. The `target` param is never used so I'm prefixing it with underscore to tell the IDE that this is intended.